recipe/meta.yaml

```
@@ -2,6 +2,8 @@
 {% set version = "0.1" %}
 {% set commit = "088985936518be7e25795a30d8ab33affa9db6ed" %}
 
+{% set nvidia_apex_proc_type = "cpu" if cuda_compiler_version == "None" else "gpu" %}
```

To fix the Windows build error, we could try the following here.

```suggestion
{% set nvidia_apex_proc_type = "cpu" if cuda_compiler_version == "None" else "gpu" %}  # [linux64]
{% set nvidia_apex_proc_type = "cpu" %}                                                # [not linux64]
```

---

@jakirkham in the recent push i made, i edited the cisupport windows build yamls to include `cuda_compiler_version:
- None` which successfully fixed the error. would this method be preferred?

---

Yeah this would be preferred as those files are automatically generated by conda-smithy.

---

@jakirkham sounds good - pushed that through

---

@jakirkham sounds good - thanks! pushed that through and it seems to be working